app/Providers/RouteServiceProvider.php

```diff
@@ -66,7 +66,7 @@ protected function mapWebRoutes()
     protected function mapApiRoutes()
     {
         if (env('APP_ENV') == "dev") {
-            Route::domain("api.vatusa.devel")
+            Route::domain("api.vatusa.hawton.dev")
```

Comment:
Let's make this domain come from ENV

Reply:
That wasn't even supposed to be in the commit.. dang it..

Reply:
Resolved in 6a45112